tests/test-cases/Group23-VIC-Machine-Service/23-05-VCH-Logs.robot

```diff
@@ -0,0 +1,56 @@
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 23-05 - VCH Logs
+Resource  ../../resources/Util.robot
+Suite Setup  Start VIC Machine Server
+Suite Teardown  Terminate All Processes  kill=True
+Test Teardown  Cleanup VIC Appliance On Test Server
+Default Tags
+
+*** Keywords ***
+Start VIC Machine Server
+    Start Process  ./bin/vic-machine-server --port 31337 --scheme http  shell=True  cwd=/go/src/github.com/vmware/vic
+    Sleep  1s  for service to start
+
+Curl No Datacenter
+    [Arguments]  ${vch-id}  ${auth}
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -s -w "\%{http_code}\n" -X GET "http://127.0.0.1:31337/container/target/%{TEST_URL}/vch/${vch-id}/log?thumbprint=%{TEST_THUMBPRINT}" -H "authorization: Basic ${auth}"
+    [Return]  ${rc}  ${output}
+
+Curl Datacenter
+    [Arguments]  ${vch-id}  ${auth}
+    ${orig}=  Get Environment Variable  TEST_DATACENTER
+    ${dc}=  Run Keyword If  '%{TEST_DATACENTER}' == '${SPACE}'  Get Datacenter Name
+    Run Keyword If  '%{TEST_DATACENTER}' == '${SPACE}'  Set Environment Variable  TEST_DATACENTER  ${dc}
+    ${dcID}=  Get Datacenter ID
```

Comment:
It doesn't look like you need any of this above this line.  And then you don't need the set environ variable below either.

Reply:
Whole keyword should just be:
Get Datacenter ID
Curl command